src/shims/ApiCompatBaseline.netfx.netstandard20.txt

```diff
@@ -0,0 +1,215 @@
+Compat issues with assembly mscorlib:
```

Comment:
it will never get built because shims.proj doesn't have a netfx configuration. They were only created because you must have forcibly passed in /t:TargetGroup=netfx when building the project.